libs/wingsdk/src/cloud/topic.md

```diff
@@ -0,0 +1,97 @@
+---
+title: cloud.Topic 
+id: topic
+description: A built-in resource for publishing messages to subscribers.
+keywords: [Wing reference, Wing language, language, Wing sdk, Wing programming language, topics]
+---
+
+The `cloud.Topic` class represents a subject of data that is open for subscription.
+
+Topics are a staple of event-driven architectures, especially those that rely on pub-sub messaging to decouple producers of data and the consumers of said data. 
+
+## Usage
+
+### Creating a topic
+
+```js
+bring cloud;
+
+let topic = new cloud.Topic();
+```
+
+### Using a topic
+
+```js
+bring cloud;
+
+let topic = new cloud.Topic();
+
+topic.onMessage(inflight (message: str) => {
+  log("Topic published message: ${message}");
+});
+```
+
+### Using a topic inflight
+The inflight api for a topic allows for publishing messages to the topic.
```

Comment:
(Thinking out loud) Maybe using old-school OO terms like "method" will reinforce the idea that these are just ordinary classes in the language
```suggestion
### Publishing to a topic

The inflight method `publish` sends a message to all of the topic's subscribers.
```